test/units/module_utils/basic/test__log_invocation.py

```diff
@@ -22,75 +22,62 @@
 
 import sys
 import json
-from io import BytesIO, StringIO
 
-from ansible.compat.six import PY3
-from ansible.utils.unicode import to_bytes
+from units.mock.procenv import swap_stdin_and_argv
```

Comment:
Not sure if this is the best way to refer to the module... what happens if we don't run nosetests form the right directory?  Maybe relative imports are what we want to use here (since the unittest code is python2.6+).
I think it's ```from ... import swap_stdin_and_argv```)